l10n_it_fatturapa_in/views/account_view.xml

```diff
@@ -45,7 +53,7 @@
                                     attrs="{'invisible': [('invoices_number', '!=', 0), ('xml_supplier_id', '!=', False), ('invoices_date', '!=', False), ('linked_invoice_id_xml', '!=', False)]}"
                                 />
                             </div>
-                            <field name="name" invisible="1" />
+                            <field name="name" />
```

Comment:
I had to remove 'invisible="1"' otherwise a lot of tests would fail because of "can't write on invisible field name".
The error is raised from https://github.com/odoo/odoo/blob/bde8ac75055d9fb53fb8b989a01f3e865afdfb27/odoo/tests/form.py#L330 which prevents writing of invisible/readonly fields to make tests consistent with web UI.
The setting of the name field is performed by fatturapa_common.py create_attachment() method.